framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceJavaParserStorage.java

```diff
@@ -607,8 +634,8 @@ private void addClass(ClassTree tree, @Nullable @BinaryName String classNameKey)
            * @param tree tree to add. Its corresponding element is used as the key for {@code
            *     classToAnnos}.
            */
-          private void addClass(ClassTree tree) {
-            addClass(tree, null);
+          private void addClass(ClassTree tree, @Nullable TypeDeclaration<?> javaParserNode) {
```

Comment:
Could you add `@param` Javadoc for this?  (I'm surprised that require-javadoc didn't require this.)